5.5/root/usr/share/container-scripts/mysql/my-paas.cnf.template

```diff
@@ -9,12 +9,15 @@ lower_case_table_names = ${MYSQL_LOWER_CASE_TABLE_NAMES}
 # The maximum permitted number of simultaneous client connections. Default: 151
 max_connections = ${MYSQL_MAX_CONNECTIONS}
 
-# The minimum length of the word to be included in a FULLTEXT index. Default: 4
+# The minimum/maximum lengths of the word to be included in a FULLTEXT index. Default: 4/20
 ft_min_word_len = ${MYSQL_FT_MIN_WORD_LEN}
-
-# The maximum length of the word to be included in a FULLTEXT index. Default: 20
 ft_max_word_len = ${MYSQL_FT_MAX_WORD_LEN}
 
 # In case the native AIO is broken. Default: 1
 # See http://help.directadmin.com/item.php?id=529
 innodb_use_native_aio = ${MYSQL_AIO}
+
+[myisamchk]
+# The minimum/maximum lengths of the word to be included in a FULLTEXT index. Default: 4/20
+ft_min_word_len = ${MYSQL_FT_MIN_WORD_LEN}
+ft_max_word_len = ${MYSQL_FT_MAX_WORD_LEN}
```

Comment:
what's the story behind supplying these twice?  are they configuring two different things in mysql?  do the values have to match in both places?